packages/terra-modal-manager/examples/index-examples/ManagerDisclosureComponent.jsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import Button from 'terra-button';
+import ContentContainer from 'terra-content-container';
+import TextField from 'terra-form/lib/TextField';
```

Comment:
With the new directory structure of examples being inside of the component package directory, you'll need to include components you use in your examples / tests as dependencies.

For example, the linter is complaining about:
```
  6:23  error  Unable to resolve path to module 'terra-form/lib/TextField'      import/no-unresolved
  6:23  error  Missing file extension for "terra-form/lib/TextField"            import/extensions
  8:26  error  Unable to resolve path to module 'terra-clinical-action-header'  import/no-unresolved
  8:26  error  Missing file extension for "terra-clinical-action-header"        import/extensions
```

To resolve, add `terra-form` and `terra-clinical-action-header` as dependencies to this component.

Reply:
Let's get this updated before we merge this. Talk to @mjhenkes or @emilyrohrbough if you need assistance resolving these lint errors.

Reply:
The linter is only freaking out because the npm install is not successfully completing. package.json has been updated with the appropriate dependencies. It should start succeeding when the DisclosureManager is merged.